cpp/src/copying/copy.cpp

```diff
@@ -20,7 +20,7 @@
 #include <cudf/column/column.hpp>
 #include <cudf/table/table.hpp>
 #include <cudf/utilities/traits.hpp>
-#include "copy_detail.hpp"
+#include <detail/copy_detail.hpp>
```

Comment:
Should this be <cudf/detail/copy_detail.hpp>?

Reply:
As per Jake's guide the path is `cudf/cpp/include/detail/`

https://github.com/rapidsai/cudf/blob/ff1b66c9e3ab260914246a7b78bd2aa42cbdb301/cpp/docs/TRANSITIONGUIDE.md#streams

Reply:
Should be `cudf/cpp/include/cudf/detail` for consistency. Jake updated guidance. :)